lib/trino-parquet/src/main/java/io/trino/parquet/predicate/ParquetLongStatistics.java

```diff
@@ -13,13 +13,13 @@
  */
 package io.trino.parquet.predicate;
 
-public class ParquetIntegerStatistics
+public class ParquetLongStatistics
         implements ParquetRangeStatistics<Long>
 {
     private final Long minimum;
     private final Long maximum;
 
-    public ParquetIntegerStatistics(Long minimum, Long maximum)
+    public ParquetLongStatistics(Long minimum, Long maximum)
```

Comment:
We may want to change the arguments to `long` in a separate commit. They are always non-null.

Reply:
This doesn't seem to be addressed, yet.